src/easier68k/assembler/assembler.py

```diff
@@ -1,10 +1,25 @@
 from ..core.util.parsing import strip_comments, has_label, get_label, strip_label, get_opcode, strip_opcode, parse_literal
-import binascii
 import sys
+import types
+import re
+import binascii
 from ..core import opcodes
+from ..core.models.list_file import ListFile
+# This *is* actually a necessary import due to using "reflection" style code further down
+# noinspection PyUnresolvedReferences
 from ..core.opcodes import *
 
 
+def for_line_stripped_comments(full_text: str):
+    line_index = 0
+    for line in full_text.splitlines():
```

Comment:
Why not use an enumerate to track the line_index?

Reply:
Didn't know that existed, that's perfect, thanks!